libr/core/panels.c

```diff
@@ -450,10 +450,12 @@ static void __toggle_window_mode(RCore *core);
 
 /* mouse */
 static bool __handle_mouse(RCore *core, RPanel *panel, int *key);
-static bool __handle_mouse_on_top(RCore *core, int x, int y);
-static void __handle_mouse_on_menu(RCore *core, int x, int y);
-static bool __handle_mouse_on_X(RCore *core, int x, int y);
+static void __handle_mouse_on_top(void *user, int x, int y);
```

Comment:
Why void, rcore is more clear

Reply:
need you to read the code and there is a reason

Reply:
Thats what i did and still i dont see any reason for that change when u are blindcasting to rcore this void* in the first line of each of those functions

Reply:
ping? ill merge when i understand this change because i  dont s ee a reason  to remove core here